### ISSUE TITLE
Shortcut joker is still straight

### ISSUE THREAD

**Issue by HappyHeads** [HeyImKyu/balatro-gay#3]
I installed the mod through modloader and the shortcut is still straight, unplayable
![2024-03-30_02-00-52-Balatro](https://github.com/HeyImKyu/balatro-gay/assets/25688381/f9a7716c-d45d-4249-ab76-6e65c7cb57b2)

**Comment by HeyImKyu** [comment]
Yeah, unfortunately the Devs are using their own localisation system wrong in some of the jokers...
I hoped they would fix it themselves but I might do a patch for this myself soon.
Is just a bit more work, cause I gotta find out which jokers are bugged this way and fix them all manually 🤷‍♀️

**Comment by morridyl** [comment]
Runner, 4 fingers, superposition and shortcut all say "straight" instead of "gay"
